techniques/system/common/1.0/rudder_stdlib_core.st

```diff
@@ -139,7 +139,7 @@ bundle agent rudder_common_report_index(technique_name, status, identifier, comp
         action    => immediate;
 
   reports:
-    full_compliance|send_reports::
+    full_compliance|reports_disabled|send_reports::
```

Comment:
I don't like writing "print a report if reports are disabled". It just doesn't make sense.

But we do need to read these reports!

I propose instead:

```
!changes_only|(changes_only.send_reports)
```